peachjam/views/documents.py

```diff
@@ -142,7 +142,9 @@ class DocumentMediaView(DetailView):
     slug_url_kwarg = "frbr_uri"
 
     def render_to_response(self, context, **response_kwargs):
-        img = get_object_or_404(self.object.images, filename=self.kwargs["filename"])
+        # there should only be one, but until we enforce uniqueness of filenames, get the first in the list
+        # see https://github.com/laws-africa/peachjam/issues/2024
+        img = get_list_or_404(self.object.images, filename=self.kwargs["filename"])[0]
```

Comment:
could this cause an index error if list is empty?

Reply:
`get_list_or_404` raises a 404 if the list is empty